features/rules/OJP/OJP000_Object-placement.feature

```diff
@@ -0,0 +1,15 @@
+@implementer-agreement
+@OJP
+@version1
+Feature: OJP000 - Object placement
+The rule verifies the presence of an object placement to define the overall context and directory of objects within the model. 
+Among others, the context definition includes default units and geometric representation context for shape representations.
+
+  @E00010
+  Scenario: Check for activation . 
+
+      Given An .IfcObject.
+      Given Its attribute .ObjectPlacement.
+      Given [Its entity type] ^is^ 'IfcLocalPlacement'
```

Comment:
```suggestion
```

I think this should be removed. OJP000 - Object placement should only check for the existence of a value for the ObjectPlacement attribute, not the type.

Otherwise it should be LOP000 - Local placement or something.

Reply:
Cool, question is though if we still need OJP000 later on if we have rules from Grid, Linear and Local placement. But for now it's good to keep it I guess.

Reply:
No, it's maybe a bit redundant .. But let's stick with having an activation rule for every functional part, to not add more exceptions to IFC, hehe :)